include/master_element/MasterElementFunctions.h

```diff
@@ -82,11 +82,11 @@ generic_grad_op(
     CoordViewType::rank == 2, "Coordinate view assumed to be rank 2");
   static_assert(OutputViewType::rank == 3, "Weight view assumed to be rank 3");
 
-  STK_ThrowAssert(
+  STK_NGP_ThrowAssert(
     AlgTraits::nodesPerElement_ == referenceGradWeights.extent(1));
-  STK_ThrowAssert(AlgTraits::nDim_ == referenceGradWeights.extent(2));
+  STK_NGP_ThrowAssert(AlgTraits::nDim_ == referenceGradWeights.extent(2));
   for (int i = 0; i < dim; ++i)
-    STK_ThrowAssert(weights.extent(i) == referenceGradWeights.extent(i));
+    STK_NGP_ThrowAssert(weights.extent(i) == referenceGradWeights.extent(i));
```

Comment:
I think the NGP string was dropped in the Trilinos we were using (15.1.1 I think)?

Reply:
Plain STK_ThrowAssert failed in my Cuda debug build. What are we using these days.

Reply:
I do have Trilinos 15.1.1.

Reply:
Oh you're right. https://github.com/Exawind/nalu-wind/pull/1255/files